build-dotnet.bat

```diff
@@ -20,19 +20,19 @@ IF ["%OSSL_LIB32_DIR%"]==[] (
 	ECHO OpenSSL could not be found
 	GOTO :QUIT
 )
-IF ["%VS120COMNTOOLS%"]==[] (
-	ECHO VS2013 could not be found
+IF ["%VS140COMNTOOLS%"]==[] (
+	ECHO VS2015 could not be found
 	GOTO: QUIT
 )
 
 :BUILD
-CALL "%VS120COMNTOOLS%VsDevCmd.bat"
+CALL "%VS140COMNTOOLS%VsDevCmd.bat"
 
 IF EXIST build RMDIR build /s /q
 MKDIR build\lib
 CD build
 
-cmake .. -DCMAKE_INSTALL_PREFIX=lib -DFULL=ON -DCMAKE_BUILD_TYPE=%CONFIGURATION% -G"Visual Studio 12 2013"
+cmake .. -DCMAKE_INSTALL_PREFIX=lib -DFULL=ON -DCMAKE_BUILD_TYPE=%CONFIGURATION% -G"Visual Studio 14 2015"
```

Comment:
The FULL option has been changed on the development branch to DNP3_ALL.

As the development branch now needs VS2015, would it be worth rebasing this?